lib/modules/dosomething/dosomething_user/dosomething_user_social_auth.inc

```diff
@@ -25,7 +25,7 @@ function dosomething_user_make_facebook_user($account_details) {
 /**
  * Registers a new user with Northstar (or upserts with existing) & returns a Drupal account
  * @param  array $account_details Account details for the user to register
- * @return $user Created (or retrieved) Drupal user
+ * @return $user
```

Comment:
This is still wrong. It can be either `@return {type}` or `@return {type} {description}`. Saying it returns a variable name doesn't really provide any value. Check out the link from my previous comment if you're still confused.